### PR TITLE
Add test for mountain peak

### PR DESCRIPTION
**Pull request by lazaa32**
Related to closed PR #335

### CODE REVIEW

**Comment by lazaa32** [comment]
To your question `That said, what specific tools functionality are you trying to test here?` this test should cover the issue with `pixel_width` we discussed here: https://github.com/openmaptiles/openmaptiles-tools/pull/334#issuecomment-720790980

File: tests/testlayers/mountain_peak/layer.sql
Comment (nyurik) [inline tests/testlayers/mountain_peak/layer.sql:4]: do you actually need this file as part of the test? I think you might as well just delete it -- doesn't appear like you are testing for any of its functionality. Or make it empty, etc.  The shorter the test, the easier it will be to debug issues that it could trigger.

File: tests/testlayers/mountain_peak/update_peak_point.sql
Comment (nyurik) [inline tests/testlayers/mountain_peak/update_peak_point.sql:1]: same for this sql file

**Review by nyurik — COMMENTED** [review]
the tests need fixing (see CI output), but otherwise looks ok. One thing - see my comment about SQL inline. Thx!

**Comment by nyurik** [comment]
So weird, somehow this PR changes the mbtiles test files so they become different in size... investigating

**Comment by nyurik** [comment]
@lazaa32 I added a filter to stop comparing on filesizes -- seems like adding extra metadata messes up generated files. See https://github.com/openmaptiles/openmaptiles-tools/pull/343/commits/17020d76782ea3d58605b92c3321893825d4d799#diff-6d15bb77fd011b2b4dca5785e7310757c4cd0134663b44ff2e741c7987b7d71aR128-R130

For the rest of the files, I just ran `make rebuild-expected` to fix expected tests.

**Comment by nyurik** [comment]
P.S. It is usually better not to push branches to the primary repo -- just use your own fork and submit a PR from it -- this way the main repo always has just the branches related to the primary project, and if someone forks it, they won't get your work-in-progress branches with it.

**Review by nyurik — APPROVED** [review]
a bit self-approving here, @lazaa32 please re-check before merging

**Comment by lazaa32** [comment]
> @lazaa32 I added a filter to stop comparing on filesizes -- seems like adding extra metadata messes up generated files. See [17020d7#diff-6d15bb77fd011b2b4dca5785e7310757c4cd0134663b44ff2e741c7987b7d71aR128-R130](https://github.com/openmaptiles/openmaptiles-tools/commit/17020d76782ea3d58605b92c3321893825d4d799#diff-6d15bb77fd011b2b4dca5785e7310757c4cd0134663b44ff2e741c7987b7d71aR128-R130)
> 
> For the rest of the files, I just ran `make rebuild-expected` to fix expected tests.

Interesting, I didn't think of that mbtiles without vacuum may differ in size.

**Comment by lazaa32** [comment]
> P.S. It is usually better not to push branches to the primary repo -- just use your own fork and submit a PR from it -- this way the main repo always has just the branches related to the primary project, and if someone forks it, they won't get your work-in-progress branches with it.

I know, I pushed this branch directly to this repo just because the maintainers collaboration didn't work on the previous PR of branch from my fork. I will delete the PR branch after merge.

**Pull request merged**
